### PR TITLE
feat(e2e): Add more networks to e2e liveness tests

### PR DESCRIPTION
**Pull request by yagopv**
## What it solves
Adding more networks to the liveness tests

### CODE REVIEW

**Comment by yagopv** [comment]
Hey @dasanra . Which networks should we add here. WDYT?

File: .github/workflows/safe-apps-check.yml [hunk 1]
```diff
@@ -39,7 +42,143 @@ jobs:
           CYPRESS_RECORD_KEY: ${{ secrets.CYPRESS_RECORD_KEY }}
           CYPRESS_BASE_URL: ${{ github.event.inputs.baseUrl || 'http://gnosis-safe.io/app' }}
           CYPRESS_NETWORK_PREFIX: ${{ github.event.inputs.networkPrefix || 'eth' }}
-          CYPRESS_TESTING_SAFE_ADDRESS: ${{ github.event.inputs.safeAddress || '0xc322cde085A4C6EFc865E77eDDdF39c31262Fc70' }}
+          CYPRESS_TESTING_SAFE_ADDRESS: ${{ github.event.inputs.safeAddress || '0xfF501B324DC6d78dC9F983f140B9211c3EdB4dc7' }}
+          CYPRESS_CONFIG_SERVICE_BASE_URL: ${{ github.event.inputs.configServiceBaseUrl || 'https://safe-client.gnosis.io' }}
```
Comment (mikhailxyz) [inline .github/workflows/safe-apps-check.yml:46]: This is not a config service URL but the client gateway senor. Also, should it use the new domain?

File: cypress/plugins/index.js [hunk 2]
```diff
@@ -33,3 +33,15 @@ module.exports = async (on, config) => {
 
   return config
 }
+
+const NETWORK_ID = {
```
Comment (mikhailxyz) [inline cypress/plugins/index.js:37]: Could we maybe also pass the chain id?

File: .github/workflows/safe-apps-check.yml (hunk 1 above)
Comment (yagopv) [reply .github/workflows/safe-apps-check.yml]: hehe yes, renamed

About the domain, web core prod is pointing to this yet 

Anyway when web-core goes live we need to redo several things here

**Review by JagoFigueroa — APPROVED** [review]

**Pull request merged**